packages/daisyui/src/templates/FieldTemplate/FieldTemplate.tsx

```diff
@@ -77,7 +77,7 @@ export default function FieldTemplate<
       uiSchema={uiSchema}
       registry={registry}
     >
-      <div className={`field-template mb-3 ${classNames || ''}`} {...divProps}>
+      <div className={`field-template mb-3${classNames || ''}`} {...divProps}>
```

Comment:
probably still need this space?
```suggestion
      <div className={`field-template mb-3 ${classNames || ''}`} {...divProps}>
```